configs/platforms/debian-9-amd64.rb

```diff
@@ -1,11 +1,3 @@
 platform "debian-9-amd64" do |plat|
-  plat.defaultdir "/etc/default"
-  plat.servicetype "systemd", servicedir: "/lib/systemd/system"
-  plat.servicetype "sysv", servicedir: "/etc/init.d"
```

Comment:
We need to keep sysv: https://github.com/puppetlabs/pxp-agent-vanagon/blob/main/configs/platforms/debian-9-amd64.rb#L4